README.md

```diff
@@ -34,17 +34,48 @@ With this compatibility library you can also use the 2.13 syntax which uses a co
 The 2.13 version consists only of an empty `scala.collection.compat` package object that allows you to write `import scala.collection.compat._` in 2.13.
 The 2.11/2.12 version has the compatibility extensions in this package.
 
-
 The library also adds backported versions of new collection types, currently `scala.collection.compat.immutable.ArraySeq`. In 2.11/2.12, this type is a new collection implementation. In 2.13, it is an alias for `scala.collection.immutable.ArraySeq`.
 
 ## Migration Tool
 
-A tool is being developed to automatically migrate code that uses the standard
-collection to use the strawman.
+We created two migration rules: 
+
+* `Collection213Upgrade` For upgrading applications (like web server, etc) from 2.11/2.12 to 2.13
+* `Collection213CrossCompat` For library that wants to cross compile to 2.11, 2.12 and 2.13
+
+```scala
+// project/plugins.sbt
+
+addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "0.7.0-RC1")
+```
+
+```scala
+// build.sbt or project/Build.scala
 
-To use it, add the [scalafix](https://scalacenter.github.io/scalafix/) sbt plugin
-to your build, as explained in
-[its documentation](https://scalacenter.github.io/scalafix/#Installation).
+// If you are using project/Build.scala add the following imports:
+import scalafix.sbt.ScalafixPlugin.autoImport.{scalafixDependencies, scalafixSemanticdb}
+
+val collectionCompatVersion = "0.2.1"
+val collectionCompat = "org.scala-lang.modules" %% "scala-collection-compat" % collectionCompatVersion
+
+libraryDependencies += collectionCompat // required for Collection213CrossCompat
```

Comment:
I think it might be more readable to have two separate sections for `Collections213Upgrade` and `Collections213CrossCompat`. It's a bit confusing which setup is required for each rule.